vochain/offchaindatahandler/census.go

```diff
@@ -9,29 +9,29 @@ import (
 )
 
 // importExternalCensus imports a census from a remote URI into the censusDB storage.
-func (d *OffChainDataHandler) importExternalCensus(uri string, data []byte) {
+func (c *OffChainDataHandler) importExternalCensus(uri string, data []byte) {
```

Comment:
What is your reasoning for using `c` instead of `d`? If it refers to chain, I must say the important concept for this name is Data.

Reply:
My first approach was checking the most common receiver name for `OffChainDataHandler`, and you are using thirteen times `c` vs three times `d` and one time `s`

Reply:
I will update it to use `d` :raised_hands:

Reply:
I will update it to use `d` :raised_hands: